std_msgs/Header header
rcp/Cpu cpu
rcp/Memory memory 
rcp/Swap swap 
rcp/Disk[] disks 
rcp/NetworkInterface[] network_interfaces
rcp/TemperatureSensor[] temperature_sensors